make/devkit/createWindowsDevkit.sh

```diff
@@ -66,13 +71,33 @@ fi
 
 # Work around the insanely named ProgramFiles(x86) env variable
 PROGRAMFILES_X86="$($WINDOWS_PATH_TO_UNIX_PATH "$(cmd.exe /c set | sed -n 's/^ProgramFiles(x86)=//p' | tr -d '\r')")"
+PROGRAMFILES="$($WINDOWS_PATH_TO_UNIX_PATH "$(cmd.exe /c set | sed -n 's/^PROGRAMFILES=//p' | tr -d '\r')")"
```

Comment:
This complex dance isn't strictly necessary; it's a special operation for just the "ProgramFiles(x86)" variable, since cygwin can't handle environment variables with `()` in the name. (See the comment above as well)

Reply:
While it did seem to work I realized that `PROGRAMFILES` wasn't converted to a unix path anymore, so I added back that conversion but without the special handling needed for the x86 variable.